src/client/actions/index.js

```diff
@@ -0,0 +1,6 @@
+export const createNewSession = (new_session) => {
+	return {
+		type: 'CREATE_NEW_SESSION',
+		payload: new_session
+	};
+};
```

Comment:
I don't see a combineActions() functions similar to how combineReducers works. If we do it the way above, we'd need to call every action as part of a greater parent object (eg. action.createNewSession) instead of just importing everything from the index file. Would this be a better idea than just defining all actions within the same index.js file?